compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/viewinterop/InteropView.uikit.kt

```diff
@@ -77,15 +74,16 @@ internal val InteropViewSemanticsKey = AccessibilityKey<UIKitInteropViewGroup>(
     }
 )
 
-private var SemanticsPropertyReceiver.interopView by InteropViewSemanticsKey
+private var SemanticsPropertyReceiver.nativeAccessibilityView by NativeAccessibilityViewSemanticsKey
 
 /**
- * Chain [this] with [Modifier.semantics] that sets the [trackInteropPlacement] of the node
- * if [enabled] is true. If [enabled] is false, [this] is returned as is.
+ * Chain [this] with [Modifier.semantics] that sets the [nativeAccessibilityView] of the node to
+ * the [interopViewGroup] if [isNativeAccessibilityEnabled] is true.
+ * If [isNativeAccessibilityEnabled] is false, [this] is returned as is.
  */
-internal fun Modifier.interopViewSemantics(enabled: Boolean, interopViewHolder: InteropViewHolder) =
-    if (enabled) {
-        this.semantics { interopView = interopViewHolder.group as UIKitInteropViewGroup }
+internal fun Modifier.interopViewSemantics(isNativeAccessibilityEnabled: Boolean, interopViewGroup: InteropWrappingView) =
```

Comment:
So, group or wrapper?

Reply:
Interop view, refactored this function entirely to make it detached from interop containment context. It's absolutely unrelated.